src/components/mobile/MobileLayout.js

```diff
@@ -43,6 +46,18 @@ const MobileLayout = () => (
       </TabPanel>
       <TabPanel>
         <p>List</p>
+        <ListEntry
+          leftIcon={<img src={LocationIcon} alt="Location Icon"></img>}
+          primaryText="Location"
+          secondaryText="Champaign, IL"
+          rightIcon={<img src={Arrow} alt="Arrow Icon"></img>}
+        />
+        <ListEntry
+          leftIcon={<img src={LocationIcon} alt="Location Icon"></img>}
+          primaryText="Location"
+          secondaryText="Champaign, IL"
+          rightIcon={<img src={Arrow} alt="Arrow Icon"></img>}
+        />
```

Comment:
Make sure to test if ListEntry works without secondaryText. Is the primaryText vertically centered?